plugins/modules/dhcp_option_space.py

```diff
@@ -0,0 +1,278 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: Infoblox Inc.
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+---
+module: dhcp_option_space
+short_description: Manage a OptionSpace
```

Comment:
```suggestion
short_description: Manage a Option Space
```

Applies for all occurences of OptionSpace

Reply:
done